packages/fx-core/scripts/download-templates-zip.js

```diff
@@ -22,6 +22,29 @@ async function step(desc, fn) {
   }
 }
 
+async function retry(id, fn, retryIntervalInMs = 5000, maxAttemptCount = 3) {
+  let exception = undefined;
+  for (let attempted = 0; attempted < maxAttemptCount; ++attempted) {
+    try {
+      if (attempted > 0) {
+        await sleep(retryIntervalInMs);
```

Comment:
maybe each fail can make the retryIntervalInMs longer

Reply:
Updated.